ccip-sdk/src/sui/index.ts

```diff
@@ -30,9 +31,10 @@ export class SuiChain extends Chain<typeof ChainFamily.Sui> {
 
   readonly network: NetworkInfo<typeof ChainFamily.Sui>
 
-  constructor() {
+  constructor(network: NetworkInfo<typeof ChainFamily.Sui>) {
```

Comment:
You don't need the constructor to use the static functions in the class. But we can leave this in until this is implemented to be actually useful